src/ketrew_backend/plugin.ml

```diff
@@ -229,8 +229,8 @@ module Long_running_implementation : Ketrew.Long_running.LONG_RUNNING = struct
           Client.get_kube_job_descriptions client [job_id]
           >>= fun l ->
           let rendered =
-            List.map l ~f:(fun (id, s) ->
-                sprintf "### Kube-Job %s\n\n%s" id  s)
+            List.map l ~f:(fun (`Id id, `Describe_output o, `Freshness frns) ->
+                sprintf "### Kube-Job %s\n### Freshness: %s\n\n\n%s" id frns o)
```

Comment:
No, it's not the "status of the job" it's what kind of result is it returning:

- `fresh`: just got the results from `kubectl`
- `old err`: “`kubectl` failed with error `err`, and the result I'm showing the is the latest successful one I know about.”